config/locales/en.yml

```diff
@@ -88,3 +88,6 @@ en:
     month: Month
     year: Year
     none: None
+  date:
+    formats:
+      full: "%B %-d, %Y"
```

Comment:
we already have date formats, including one named full. Is it reasonable in the code to reuse those, or move this to the section where they are?

Reply:
I have seen only the time format named full. 
When is called 
`created_date: I18n.l(Date.today, format: :full, default: nil)`
He looks for that one I have inserted.